src/improvements/tasks/sep/005-deputy-pause-key-rotation/config.toml

```
@@ -0,0 +1,6 @@
+l2chains = [{name = "OP Sepolia Testnet", chainId = 11155420}] # e.g. [{name = "OP Mainnet", chainId = 10}]
+templateName = "DeputyPauseRotationKey"
```

naming nit, a bit tedious to rename the contract/files but this better matches the folder name and is more clear
```suggestion
templateName = "DeputyPauseKeyRotation"
```

---

DeputyPauseKeyRotationTemplate